src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const isSelectedTabIdCorrect = tabs
+    .some(tab => tab.id === selectedTabId) ? selectedTabId : tabs[0].id;
+
+  const selectedTab = tabs.find(tab => tab.id === isSelectedTabIdCorrect);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={cn({ 'is-active': tab.id === isSelectedTabIdCorrect })}
+              data-cy="Tab"
+            >
+              <a
+                onClick={() => {
+                  if (tab.id !== isSelectedTabIdCorrect) {
+                    onTabSelected(tab);
+                  }
```

Comment:
I think, better create function if you use some condition

Reply:
➕ , let's create a separate function to handle this click